install.sh

```diff
@@ -9,62 +9,61 @@ Red='\033[0;31m'          # Red
 Blue='\033[0;34m'         # Blue
 
 need_cmd () {
-    if ! command -v "$1" > /dev/null 2>&1;then
-        printf "${Red}need '$1' (command not found)${Color_off}\n"
-        exit 0
+    if ! hash "$1" &>/dev/null; then
+        echo "${Red}need '$1' (command not found)${Color_off}"
```

Comment:
![2017-01-05_1362x555](https://cloud.githubusercontent.com/assets/13142418/21648816/491060d8-d2da-11e6-88f9-ea614b6008e9.png)

@mhinz thanks for PR, I think we should use `printf` to make colored output.